front/app/components/FormBuilder/edit/index.tsx

```diff
@@ -351,7 +351,9 @@ const FormEdit = ({
       );
     } catch (error) {
       const errorType =
-        error?.errors?.form[0].error === 'stale_data'
+        Array.isArray(error?.errors?.form) &&
```

Comment:
Similar to https://github.com/CitizenLabDotCo/citizenlab/pull/10456

Reply:
`error?.errors?.form[0]` can be `undefined`. In that case, a TypeError is raised and the remainder of the function
```
  handleHookFormSubmissionError(error, setError, errorType);
  setIsSubmitting(false);
```
 is not executed.